### DISCUSSION TITLE
Hide virtual camera when not in use

### DISCUSSION THREAD

**Discussion by yigitkerem** [obsproject/obs-studio#3989] [General]
I don't think I am the only one who is annoyed by the OBS Virtual Camera option popping up and sometimes automatically going default even while we are not using OBS.

According to my testing (on macOS): When we remove the plugin from /Library/CoreMediaIO/Plug-Ins/DAL/, the option goes away without restart so automatically deleting that folder when we close the virtual camera might be a quick fix.

The feature works great, but the virtual camera still appearing even though OBS being closed annoys me, especially when some apps default to the OBS Virtual Cam

**Comment by sebastian-s-beckmann** [comment]
I feel the complete different way.
Removing the virtual camera, both on Windows and Mac, means that it needs to be reinstalled before using it again.

(The following workflow is the Mac way, but I'm pretty sure its the same / similar on Windows)
The camera would need to first be started via OBS, then the application you want would need to be restarted, and only then you could use the virtual camera.
Installing the camera requires admins privileges every time, and so would uninstalling.

This workflow would be extremely annoying to most people, especially since restarting the browser, one of the common use cases, would disrupt anyones work.
